src/app/core/views/Preferences/tabs/Account/Language.tsx

```diff
@@ -25,8 +25,8 @@ export default function Language(): JSX.Element {
 
   useEffect(() => {
     if (localStorageLanguage) {
-      setCurrentLangText(currentLang[localStorageLanguage as string]);
-      setLang(localStorageLanguage as string);
+      setCurrentLangText(currentLang[localStorageLanguage]);
+      setLang(localStorageLanguage);
```

Comment:
If you always have to change both states at the same time, add it to a function, I see that the code is repeated 3 times.